packages/aws-cdk-lib/aws-ec2/lib/vpc.ts

```diff
@@ -1388,11 +1444,18 @@ export class Vpc extends VpcBase {
    */
   private readonly resource: CfnVPC;
 
+  /**
+   * The VPC protocol
+   */
+  private readonly vpcProtocol: VpcProtocol;
```

Comment:
Especially if we're going to add IPV6_ONLY, I think easier to work with will be:

```ts
private readonly useIpv4: boolean;
private readonly useIpv6: boolean;
```